linebot/webhook.py

```diff
@@ -267,11 +267,16 @@ def __add_handler(self, func, event, message=None):
 
     @staticmethod
     def __get_args_count(func):
+        HANDLER_ARGSIZE_MAXIMAM = 2
```

Comment:
[imo]
How about returning the flag that function has varadic arguments or not in addition to the number of non-varadic arguments?

In this way, we should modify https://github.com/line/line-bot-sdk-python/blob/ec635ca981004585bad46f5163dde1a1a2420ab2/linebot/webhook.py#L256-L262
 to following, but I think that it is more clear.
```python
has_varargs, args_count = self.__get_args_count(func)
if has_varargs or args_count >= 2:
    func(event, payload.destination)
elif args_count == 1:
    func(event)
else:
    func()
```